daemons/maap/windows/src/maap_log_windows.c

```diff
@@ -85,7 +85,13 @@ static int loggingThreadRunning = FALSE;
 DWORD WINAPI loggingThreadFn(LPVOID pv);
 static HANDLE loggingThread = NULL;
 
-#define THREAD_STACK_SIZE									65536
+#if !defined(PTHREAD_STACK_MIN)
+#error "PTHREAD_STACK_MIN variable not defined"
```

Comment:
@msutar, on Windows, I think, PTHREAD_STACK_MIN is defined in pthread.h. This header file is not included.